api/src/datasets/datasets.controller.ts

```diff
@@ -1,61 +1,91 @@
-import type { DatasetInfo } from '@databank/types';
 import { ParseIdPipe } from '@douglasneuroinformatics/nestjs/core';
-import { Body, Controller, Delete, Get, Param, Patch, Post, Query } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Param, Patch, Post, UploadedFile, UseInterceptors } from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
 import { ApiOperation, ApiTags } from '@nestjs/swagger';
-import { type ObjectId } from 'mongoose';
+import 'multer';
```

Comment:
What is the point of this? Is there some kind of side effect happening here (don't see in NestJS docs)? If so, I would move to either `AppModule` or `main.ts` since all side effects should be defined in the same place.